apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-header/platform-date-picker-header.component.html

```diff
@@ -0,0 +1,21 @@
+<header>Date Picker</header>
+<description>
+    Platform DatePicker is an extension of the Date Picker component present in
+    <a href="https://sap.github.io/fundamental-ngx/#/core/datePicker">fundamental-ngx/core</a> to work with
+    <a href="https://sap.github.io/fundamental-ngx/#/platform/form">Platform Form</a>. <br />
+    Platform DatePicker keeps the same property and behaviour with core DatePicker.
+    <br />
+    <br />
+    <import module="PlatformDatePickerModule"></import>
+    <br />
```

Comment:
It worths to add that platform datepicker mogule relies on DatetimeAdapter provider

Reply:
added information regarding DatetimeAdapter provider